x-pack/plugins/security_solution/common/detection_engine/rule_management/api/urls.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { INTERNAL_DETECTION_ENGINE_URL } from '../../../constants';
+
+export const RULES_INFO_URL = `${INTERNAL_DETECTION_ENGINE_URL}/rules/_info`;
```

Comment:
"Rules info" seems to be a very generic term and conveys little information on what that endpoint is about. As it is an internal endpoint designed to be used exclusively for the rules management table, I propose renaming it to something like `rules/_rule_management_filters` and all associated types/methods accordingly.